app/views/projects/samples/attachments/_table.html.erb

```diff
@@ -4,17 +4,21 @@
       <thead class="text-slate-700 bg-slate-50 dark:bg-slate-900 dark:text-slate-400">
         <tr>
           <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.filename") %></th>
+          <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.format") %></th>
+          <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.type") %></th>
           <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.size") %></th>
           <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.created_at") %></th>
           <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.action") %></th>
         </tr>
       </thead>
       <tbody id="attachments-table-body">
         <% attachments.each do |attachment| %>
```

Comment:
**suggestion:** Make this a partial and use it in the create turbo response.  Pass in the persistent attachments to it in the create method on the attachments_controller.  You will have to keep track of which ones are persisted and which ones failed, you can do a filter to find out which ones persisted.